README.md

```
@@ -1,6 +1,146 @@
-# backend
+# 한화시스템 Beyond SW 21기 최종 프로젝트 - Rhight
 
-## Requirements
+---
+# 😄 팀소개 Re:verse
+|[김세현](https://github.com/sekong11)|[양승재](https://github.com/DMXZC)|[이상준](https://github.com/Ongsaem0)|[정규원](https://github.com/Gyuwon-Jung)|
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)